### PR TITLE
Add meeting notes for December 2021

### PR DESCRIPTION
**Pull request by rprouse**

### CODE REVIEW

File: Meetings/2021-12-08-meeting-minutes.md [hunk 1]
```diff
@@ -0,0 +1,49 @@
...
+- Calvin A. Allen - @CalvinAllen
+- Dan Siegel - @dansiegel
+- James Newton-King - @JamesNK
+- James Turner - @Turnerj
+- Pascal Berger - @pascalberger
+- Nils Andresen - @nils-a
+
+## Recording
+
+**TBD**
```
Comment (nils-a) [inline Meetings/2021-12-08-meeting-minutes.md:16]: @rprouse Not sure if this is a "good" place to discuss, but:
The recording for the last (second to last? Well, the "other" meeting..) are also still "TBD". Is that a "need the time to upload" issue, or rather a "don't know where to put it" issue? 
Is there anything someone can do to help?

File: Meetings/2021-12-08-meeting-minutes.md [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+
+- Review Action Items
+- Discuss the process to leave the foundation
+
+## Notes
+
+### Process for Leaving the Foundation
+
+- Reviewed the process to leave the Foundation as posted in the [GitHub discussion](https://github.com/dotnet-foundation/Home/discussions/68#discussioncomment-1774246).
+- After all comments have been reviewed, the board will meet with the lawyers to reverse the contracts that have been put in place, the board will work with the projects individually that have sent notices to leave.
```
Comment (nils-a) [inline Meetings/2021-12-08-meeting-minutes.md:34]: This sentence sounds funny to me. (Could well be my English, though.)

> [...] the board will meet with the lawyers to reverse the contracts that have been put in place [...]

Sounds a bit like "all of the contracts". Maybe something like: 

```suggestion
- After all comments have been reviewed, the board will meet with the lawyers to reverse the contracts of the projects that have sent notices to leave. The board will work with each of the aforementioned projects individually.
```

**Review by nils-a — COMMENTED** [review]
LGTM, one question and one clarification.

File: Meetings/2021-12-08-meeting-minutes.md (hunk 1 above)
Comment (rprouse) [reply Meetings/2021-12-08-meeting-minutes.md:16]: It is a find a don't know where to store it issue. I tried sharing from the foundation OneDrive but external sharing is blocked. I want to get that changed. We also discussed YouTube, but I'm not sure meeting recordings should be in the main Foundation YouTube feed.

Comment (Turnerj) [reply Meetings/2021-12-08-meeting-minutes.md:16]: Could do an unlisted video and just link it here? Wouldn't appear in anyone's subscription feeds then.

Comment (rprouse) [reply Meetings/2021-12-08-meeting-minutes.md:16]: I'll ask about that, thanks. I'm not going to hold up merging this though, so I'm going to resolve.

**Pull request merged**